valhalla/mjolnir/landmark_builder.h

```diff
@@ -0,0 +1,129 @@
+  static const std::unordered_map<std::string, LandmarkType> string_to_landmark_type =
+      {{"restaurant", LandmarkType::restaurant},
+       {"school", LandmarkType::school},
+       {"fast_food", LandmarkType::fast_food},
+       {"cafe", LandmarkType::cafe},
+       {"fuel", LandmarkType::fuel},
+       {"bank", LandmarkType::bank},
+       {"pharmacy", LandmarkType::pharmacy},
+       {"kindergarten", LandmarkType::kindergarten},
+       {"bar", LandmarkType::bar},
+       {"hospital", LandmarkType::hospital},
+       {"post_office", LandmarkType::post_office},
+       {"pub", LandmarkType::pub},
+       {"clinic", LandmarkType::clinic},
+       {"police", LandmarkType::police},
+       {"fire_station", LandmarkType::fire_station},
+       {"car_wash", LandmarkType::car_wash},
+       {"charging_station", LandmarkType::charging_station},
+       {"college", LandmarkType::college},
+       {"university", LandmarkType::university},
+       {"theatre", LandmarkType::theatre},
+       {"cinema", LandmarkType::cinema}};
+
+  auto it = string_to_landmark_type.find(s);
+  if (it != string_to_landmark_type.cend()) {
+    return it->second;
+  }
+  return LandmarkType::null;
+}
+
+inline std::string landmark_type_to_string(LandmarkType t) {
+  static const std::unordered_map<uint8_t, std::string> landmark_type_to_string =
+      {{static_cast<uint8_t>(LandmarkType::null), "na"},
+       {static_cast<uint8_t>(LandmarkType::restaurant), "restaurant"},
+       {static_cast<uint8_t>(LandmarkType::school), "school"},
+       {static_cast<uint8_t>(LandmarkType::fast_food), "fast_food"},
+       {static_cast<uint8_t>(LandmarkType::cafe), "cafe"},
+       {static_cast<uint8_t>(LandmarkType::fuel), "fuel"},
+       {static_cast<uint8_t>(LandmarkType::bank), "bank"},
+       {static_cast<uint8_t>(LandmarkType::pharmacy), "pharmacy"},
+       {static_cast<uint8_t>(LandmarkType::kindergarten), "kindergarten"},
+       {static_cast<uint8_t>(LandmarkType::bar), "bar"},
+       {static_cast<uint8_t>(LandmarkType::hospital), "hospital"},
+       {static_cast<uint8_t>(LandmarkType::post_office), "post_office"},
+       {static_cast<uint8_t>(LandmarkType::pub), "pub"},
+       {static_cast<uint8_t>(LandmarkType::clinic), "clinic"},
+       {static_cast<uint8_t>(LandmarkType::police), "police"},
+       {static_cast<uint8_t>(LandmarkType::fire_station), "fire_station"},
+       {static_cast<uint8_t>(LandmarkType::car_wash), "car_wash"},
+       {static_cast<uint8_t>(LandmarkType::charging_station), "charging_station"},
+       {static_cast<uint8_t>(LandmarkType::college), "college"},
+       {static_cast<uint8_t>(LandmarkType::university), "university"},
+       {static_cast<uint8_t>(LandmarkType::theatre), "theatre"},
+       {static_cast<uint8_t>(LandmarkType::cinema), "cinema"}};
+
+  auto it = landmark_type_to_string.find(static_cast<uint8_t>(t));
+  if (it != landmark_type_to_string.cend()) {
+    return it->second;
+  }
+  return "null";
+}
```

Comment:
i know i said we would need this eventually but we dont need it yet right? on second thought i think we will never need it because these things will need to be localized to be used in directions like "turn left at the gas station". we have a wholly other way of getting those types of things into the response so i think we can remove this unless there is some other use?
```suggestion
```

Reply:
you are right, we don't need it yet. will delete it. 
how do you plan to do the wholly other way?

Reply:
its a large topic, once we have this stuff in the tiles we can go over it. if you are interested now you can go look at https://github.com/valhalla/valhalla/blob/master/locales/README.md for a start